feat: add rescue mission phase 1 aip (#299)
aave/aip: content/aips/RESCUE-MISSION-PHASE-1-LONG.md

---
title: Rescue Mission Phase 1 Long Executor

> author: BGD Labs (@bgdlabs)

Reviewer: ```suggestion
author: BGD Labs (@bgdlabs), proposer ACI (@aavechan)
```

> shortDescription: Deploy contracts for the phase 1 of the rescue mission that need the Long Executor

Reviewer: ```suggestion
shortDescription: Activation of Aave rescue mission Phase 1, Level 2 (Long) Executor part
```

discussions: https://governance.aave.com/t/bgd-rescue-of-tokens-locked-on-aave-overview-and-phase-1/8150/1
created: 2023-02-22
---

## Simple Summary
A Proposal that implements and deploys the necessary smart contracts to execute Phase 1 of the rescue mission.

## Motivation

> Since the birth of the Aave ecosystem (even on LEND times) as it usually happens on crypto, a lot of people have sent tokens by mistake to incorrect addresses.

Reviewer: ```suggestion
Since the birth of the Aave ecosystem, multiple people have sent tokens by mistake to incorrect addresses.
```

- Users who sent [LEND](https://etherscan.io/token/0x80fB784B7eD66730e8b1DBd9820aFD29931aab03) to the [LEND](https://etherscan.io/token/0x80fB784B7eD66730e8b1DBd9820aFD29931aab03) smart contract
- User who sent [LEND](https://etherscan.io/token/0x80fB784B7eD66730e8b1DBd9820aFD29931aab03) to the [LendToAaveMigrator](https://etherscan.io/address/0x317625234562B1526Ea2FaC4030Ea499C5291de4) smart contract
- Users who sent [AAVE](https://etherscan.io/token/0x7fc66500c84a76ad7e9c93437bfc5ac33e2ddae9) and [stkAAVE](https://etherscan.io/token/0x4da27a545c0c5b758a6ba100e3a049001de870f5) to the [stkAAVE](https://etherscan.io/token/0x4da27a545c0c5b758a6ba100e3a049001de870f5) contract

The LEND sent to the specified contracts will be claimed already as AAVE tokens with the transformation LEND to AAVE already taken into account (1 AAVE = 100 LEND)

## Specification

> To rescue the tokens specified on Phase 1 that are on AAVE and stkAAVE token contracts, the payload will deploy and initialize the new implementations for the AAVE token and the stkAAVE token with appropriate amounts and send them to the AaveMerkleDistributor specified on the Rescue Short Executor Proposal.

Reviewer: ```suggestion
To rescue the tokens specified on Phase 1 that are on AAVE and stkAAVE token contracts, the payload will connect and initialize the new implementations for the AAVE token and the stkAAVE token with appropriate amounts and send them to the AaveMerkleDistributor specified on the Rescue Short Executor Proposal.
To be as less invasive as possible, these new implementations only include that extra logic on their `initialize()` function, with everything else remaining the same.
```
